### PR TITLE
Basic example implementation

### LINKED ISSUE #2: example(s)
more fun to maintain than a readme xD

### PR DESCRIPTION
**Pull request by Apercu**
Little contribution for a live demo of the component.
Add budo for dev, build script and deploy to github pages utility.

You can have a look on this [here](http://apercu.github.io/deku-sheet/)

Closes #2.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,6 +5,18 @@ lib: node_modules
 	$(BIN)/babel --out-dir $@ src --source-maps inline
 	$(BIN)/postcss --use autoprefixer --dir $@ src/index.css
 
+build:
+	mkdir -p build
+	cp site/* build
+	cp src/index.css build
+	${BIN}/browserify site/index.js -o build/index.js -t [ babelify --presets [ es2015 ] ]
```
Comment (stephenmathieson) [inline Makefile]: shouldn't need `--presets` since they're listed in our babelrc

**Comment by stephenmathieson** [comment]
lgtm

thoughts @stevenmiller888?

**Comment by stevenmiller888** [comment]
lgtm, thanks @Apercu !

File: Makefile (hunk 1 above)
Comment (Apercu) [inline Makefile]: Sorry, just got home and tested it without the presets, it doesn't seems to pickup the config of the `babelrc`, so we need those two I think.

**Comment by Apercu** [comment]
Happy to help @stevenmiller888 and @stephenmathieson! You might want to run `make build && make deploy` to create the build and publish it so you can have the http://segmentio.github.io/deku-sheet url :smiley:

**Comment by stevenmiller888** [comment]
@Apercu Done!

**Comment by Apercu** [comment]
Nice!

**Pull request merged**